cmd/qat_plugin/qat_plugin.go

```diff
@@ -208,6 +211,25 @@ func isValidVfDeviceID(vfDevID string) bool {
 	return false
 }
 
+func swapBDF(devstrings []string) (result []string) {
+	for _, dev := range devstrings {
+		tmp := strings.Split(dev, ":")
+		result = append(result,
+			fmt.Sprintf("%v:%v:%v", tmp[2], tmp[1], tmp[0]))
+	}
+	return
+}
```

Comment:
Since the length of `devstrings` is known it makes sense to avoid memory realloc's on every iteration. Like in the code
```go
func swapBDF(devstrings []string) []string {
    result := make([]string, len(devstring))
    for n, dev := range devstrings {
        tmp := string.Split(dev, ":")
        result[n] = fmt.Sprintf("%v:%v:%v", tmp[2], tmp[1], tmp[0])
    }
    return result
}
```

Reply:
yes, of course